src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+---
+layout: post
+title: Monitor your web application with the Reporting API
+subhead: |
+  Use the Reporting API to monitor your site and get security reports, deprecation reports, and more.
+authors:
+  - maudn
+date: 2021-06-14
+updated: 2021-06-14
+description: |
+  Use the Reporting API to monitor your site and get security reports, deprecation reports, and more.
+hero: image/O2RNUyVSLubjvENAT3e7JSdqSOx1/PEgnzZFQVPhP2PyOnMm8.jpg
+alt: |
+  A person looking into a scope, symbolizing monitoring and observation via the Reporting API.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - monitoring
+  - security
+---
+
+{% Banner 'caution', 'body' %} This is an API guide with detailed usage examples for the
+**Reporting API (v1)**, which uses the `Reporting-Endpoints` header. If you're already
+using the legacy Reporting API with the `Report-To` header and need to migrate to the new
+version (v1), head over to the [migration post](/reporting-api-migration) instead. {%
+endBanner %}
+
+Some errors only occur in production. You never see them locally or during development
+because **real users**, **real networks**, and **real devices** change the game. The
+Reporting API helps catch some of these errors across your site and transmit them to an
+endpoint you've specified.
+
+This API enables you to activate a reporting mechanism that's **operated by the browser**.
+When the Reporting API configured on your site, **reports can be generated** when certain
+policies are violated (or when deprecated APIs are used, when the browser intervenes on
+your site or crashes, and more). The browser then sends those report(s) to the
+**endpoint(s) i.e. report collectors** you've configured.
+
+Setting up the Reporting API gives you peace of mind that when users experience these
+types of unforeseen errors, you'll be aware and able to fix them if needed. ✨
+
+```http
+  Reporting-Endpoints: main-endpoint="https://reports.example/main", default="https://reports.example/default"
+  Content-Security-Policy: script-src 'self'; object-src 'none'; report-to main-endpoint;
+```
+
+This post covers what this API can do, how it works, and how you can use it.
+
+Let's dive in!
+
+## Demo and code
+
+{% Banner 'info', 'body' %}
+
+💥 [Demo site: report generation page](https://reporting-api-demo.glitch.me). This page
+uses the new Reporting API via the `Reporting-Endpoints` header. See the
+[code](https://glitch.com/edit/#!/reporting-api-demo).
+
+📬 [Demo site: reporting endpoint](https://reports-endpoint.glitch.me). This server
+receives and displays reports. See the
+[code](https://glitch.com/edit/#!/reports-endpoint).
+
+{% endBanner %}
```

Comment:
```suggestion

💥 [Demo site: report generation page](https://reporting-api-demo.glitch.me). This page
uses the new Reporting API via the `Reporting-Endpoints` header. See the
[code](https://glitch.com/edit/#!/reporting-api-demo).

📬 [Demo site: reporting endpoint](https://reports-endpoint.glitch.me). This server
receives and displays reports. See the
[code](https://glitch.com/edit/#!/reports-endpoint).

```

Reply:
Removing the banner for readability here too. I'm torn about the emoji, they are whimsical but for screen readers it's just clutter. It reads as "Explosion demo site..." and more jarringly, "Open letterbox with raised flag demo site..."